pkg/reader/ccip.go

```diff
@@ -851,6 +851,23 @@ func (r *ccipChainReader) LinkPriceUSD(ctx context.Context) (cciptypes.BigInt, e
 	return linkPriceUSD, nil
 }
 
+func (r *ccipChainReader) GetDataAvailabilityConfig(ctx context.Context, srcChain cciptypes.ChainSelector) (
+	destDAOverheadGas, destGasPerDAByte, destDAMultiplierBps int64, err error) {
```

Comment:
This is super EVM specific, whats the plan for non-EVM?

Reply:
Either non-EVM chains will need to expose cost params or there needs to be a cost-estimation service we can call. We should have a plan for this, but we should move forward with this in the meantime.